queries.sql

```diff
@@ -18,6 +18,41 @@ SELECT * from animals WHERE name <> 'Gabumon';
 
 SELECT * from animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
 
+Begin;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=to%20unspecified.-,Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.,-Inside%20a%20transaction), Please verify that change was made and verify that the species columns went back after the rollback.

Hint: 
```sql
   BEGIN; 
   UPDATE animals
   SET species = 'unspecified'; 
   SELECT species from animals; -- verify that change was made
   ROLLBACK;
   SELECT species from animals; -- verify that change was undone
```

Reply:
- [ ] Kindly implement the suggested change by the previous reviewer to verify that the change was made and verify that the species columns went back after the rollback.

e.g
```
   BEGIN; 
   UPDATE animals
   SET species = 'unspecified'; 
   SELECT species from animals; -- verify that change was made
   ROLLBACK;
   SELECT species from animals; -- verify that change was undone
```